node/cmd/guardiand/node.go

```diff
@@ -893,110 +818,80 @@ func runNode(cmd *cobra.Command, args []string) {
 		logger.Fatal("failed to load guardian key", zap.Error(err))
 	}
 
-	guardianAddr := ethcrypto.PubkeyToAddress(gk.PublicKey).String()
```

Comment:
This seems like a simplification that would have been fine to keep so that you don't do 
```
ethcrypto.PubkeyToAddress(gk.PublicKey)
```
everywhere.